cmd/main.go

```diff
@@ -33,9 +33,12 @@ func init() {
 }
 
 var (
-	endpoint          = flag.String("endpoint", "unix:/tmp/csi.sock", "CSI endpoint")
-	gceConfigFilePath = flag.String("cloud-config", "", "Path to GCE cloud provider config")
-	vendorVersion     string
+	endpoint             = flag.String("endpoint", "unix:/tmp/csi.sock", "CSI endpoint")
+	cloudConfigFilePath  = flag.String("cloud-config", "", "Path to GCE cloud provider config")
+	runControllerService = flag.Bool("run-controller-service", true, "If set to false then the CSI driver does not activate its controller service (default: true)")
+	runIdentityService   = flag.Bool("run-identity-service", true, "If set to false then the CSI driver does not activate its identity service (default: true)")
```

Comment:
sorry for the churn - on second though, i don't think we need the ability to turn off the identity server since anything using the driver is most likely going to have to be able to interrogate the driver for at least its name.